README.md

```diff
@@ -42,10 +41,69 @@ Turn Ton logging to stderr. Default to false.
 ### `loglevel`
 Defines the logging levels. Default to info. Valid values are: debug, info, warn, error
 
+### Notes:
```

Comment:
I have added the notes here. Looking forward to your feedback 🙇🏼

Reply:
This is too long. The note section is supposed to note the things people unexpected to find, and also supposed to be concise. In the http get case, it is about the field value matches the default value won't be presented in url params. and a short example like the following will be enough . 

`{fieldA: "A", fieldB: "" fieldC:1}` will turned in the `/path?filedA=A&fieldC=1`

Reply:
Hmm... ok I will try my best to make it more concise. Thank you for the feedback 🙏🏼

Reply:
Sorry for the delay. I have simplified the notes section 🙇🏼